resources/js/Pages/Dashboard.jsx

```diff
@@ -1,30 +1,67 @@
-import AuthenticatedLayout from '@/Layouts/AuthenticatedLayout';
-import { Head } from '@inertiajs/react';
-import SecondaryButton from '@/Components/SecondaryButton';
+import AuthenticatedLayout from "@/Layouts/AuthenticatedLayout";
+import { Head } from "@inertiajs/react";
+import gear from "../../assets/icons/gear.svg";
+import gift from "../../assets/icons/gift.svg";
+import heart from "../../assets/icons/heart.svg";
+import lightbulb from "../../assets/icons/lightbulb.svg";
+import PrimaryButton from "@/Components/PrimaryButton";
 
 export default function Dashboard() {
+    const dashboardItems = [
+        {
+            image: gift,
+            title: "gift exchanges",
+            link: "/dashboard",
+        },
+        {
+            image: heart,
+            title: "hobbies",
+            link: "/dashboard",
+        },
+        {
+            image: lightbulb,
+            title: "ideas",
+            link: "/dashboard",
+        },
+        {
+            image: gear,
+            title: "settings",
+            link: "/dashboard",
+        },
+    ];
     return (
-        <AuthenticatedLayout
-            header={
-                <h2 className="text-xl font-semibold leading-tight text-gray-800 dark:text-gray-200">
-                    Dashboard
-                </h2>
-            }
-        >
+        <AuthenticatedLayout>
             <Head title="Dashboard" />
-
-            <SecondaryButton className="ms-4">
-            Test
-            </SecondaryButton>
-            <div className="py-12">
-                <div className="mx-auto max-w-7xl sm:px-6 lg:px-8">
-                    <div className="overflow-hidden bg-white shadow-sm sm:rounded-lg dark:bg-gray-800">
-                        <div className="p-6 text-gray-900 dark:text-gray-100">
-                            You're logged in!
-                        </div>
-                    </div>
-                </div>
-            </div>
+            <h1 className="sr-only">Dashboard</h1>
+            <h2 className="leading-tight text-black font-bold text-5xl tracking-[-.005em] font-baskerville">
+                Hello, RICHARD
```

Comment:
Update so it uses the user name dynamically instead of static value

Reply:
Resolved. Used code posted by @choir241 on Discord chat